### PR TITLE
Fix [Dark Mode] UI inconsistency user selects autofill values

### PR DESCRIPTION
**Pull request by Dvinod1**
Issue #6928 

> changes made in these below files  to solve the issue.

- changes made in frontend/src/_styles/theme.scss
- changes made in frontend/src/_styles/tabler.scss

### CODE REVIEW

**Review by stepinfwd — CHANGES_REQUESTED** [review]
Please revert changes on `package-lock` , `tabler.scss`, `package.json`

**Comment by Dvinod1** [comment]
> 

Can I get any help about this, I can't able to solve, I am a little bit confused.

**Comment by stepinfwd** [comment]
> > 
> 
> Can I get any help about this, I can't able to solve, I am a little bit confused.

we only need changes related to this PR that would be only in `theme.scss` revert all other changes we dont want to make changes in package.json and package lock etc so try to copy paste these files from develop  and check if its resolved

File: frontend/package.json [hunk 1]
```diff
@@ -205,4 +205,4 @@
       "jsx"
     ]
   }
-}
```
Comment (stepinfwd) [inline frontend/package.json:208]: please remove space this too @Dvinod1

Comment (Dvinod1) [reply frontend/package.json:208]: I am not understanding, Please help me?

**Comment by stepinfwd** [comment]
https://github.com/ToolJet/ToolJet/assets/31006028/cfe46cdb-ff21-4d47-9643-b90652a55d62

Hi @Dvinod1 , you can see a flash while hovering on suggested list please try to remove this

**Review by Dvinod1 — COMMENTED** [review]
solved the flash problem while hovering

**Comment by kavinvenkatachalam** [comment]
@Dvinod1 Can you revert the `package.json` changes from the PR?

**Comment by Dvinod1** [comment]
> @Dvinod1 Can you revert the `package.json` changes from the PR?

I am not understanding how to do it, can you help me ? Please.

**Comment by stepinfwd** [comment]
> > @Dvinod1 Can you revert the `package.json` changes from the PR?
> > @kavinvenkatachalam
> > I am not understanding how to do it, can you help me ? Please.

There is only an extra space in package.json just remove that 

> > @Dvinod1 Can you revert the `package.json` changes from the PR?
> > @kavinvenkatachalam
> > I am not understanding how to do it, can you help me ? Please.

There is some extra space in package.json file just remove that ,  finally in this PR there should be change only in theme.scss

**Review by Dvinod1 — COMMENTED** [review]
I have removed the changes in package.json

**Review by stepinfwd — DISMISSED** [review]

**Comment by stepinfwd** [comment]
@Dvinod1  please resolve conflicts

**Comment by stepinfwd** [comment]
@Dvinod1  Please resolve the conflicts we want to get this PR merged

**Comment by Dvinod1** [comment]
I have resolved the conflict , please check

**Review by kavinvenkatachalam — APPROVED** [review]

**Comment by Dvinod1** [comment]
Wow, Finally my First PR got merged feeling good, 
Thanks to all and many more to come, learned a lot.

**Pull request merged**